### DISCUSSION TITLE
[QUESTION] Pulsar adapter for ogg

### DISCUSSION THREAD

**Discussion by dik111** [apache/pulsar#19995] [General]
Hi all:  
I saw an article by Lakala on the Internet saying that messages can be obtained from OGG(Oracle Golden Gate) through the pulsar adapter. Is there any plan to develop this adapter in the future?
[对 Kafka 和 Pulsar 进行性能测试后，拉卡拉将消息平台统一换成了 Pulsar](https://www.jianshu.com/p/5cd80bb3d401)

**Comment by wolfstudy** [comment]
> Hi all:
> I saw an article by Lakala on the Internet saying that messages can be obtained from OGG(Oracle Golden Gate) through the pulsar adapter. Is there any plan to develop this adapter in the future?
> [对 Kafka 和 Pulsar 进行性能测试后，拉卡拉将消息平台统一换成了 Pulsar](https://www.jianshu.com/p/5cd80bb3d401)

Hello @dik111 This is a good idea. If the OGG adapter is supported, it will be convenient for this part of users to use Pulsar better.

We can start with this issue to see if there are students familiar with OGG in the community, and we can support this feature as soon as possible. If you are interested in this feature, you are always welcome to contribute this feature back to Pulsar.
